packages/scaffold/src/views/w3m-update-email-wallet-view/index.ts

```diff
@@ -0,0 +1,100 @@
+import { ConnectorController } from '@web3modal/core'
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { state } from 'lit/decorators.js'
+import { ref, createRef } from 'lit/directives/ref.js'
+import type { Ref } from 'lit/directives/ref.js'
+import styles from './styles.js'
+import { SnackController, RouterController } from '@web3modal/core'
+
+@customElement('w3m-update-email-wallet-view')
+export class W3mUpdateEmailWalletView extends LitElement {
+  public static override styles = styles
+
+  // -- Members ------------------------------------------- //
+  private formRef: Ref<HTMLFormElement> = createRef()
+
+  private initialValue = RouterController.state.data?.email ?? ''
+
+  // -- State & Properties -------------------------------- //
+  @state() private email = ''
+
+  @state() private loading = false
+
+  public override firstUpdated() {
+    this.formRef.value?.addEventListener('keydown', event => {
+      if (event.key === 'Enter') {
+        this.onSubmitEmail(event)
+      }
+    })
+  }
+
+  // -- Render -------------------------------------------- //
+  public override render() {
+    const showSubmit = !this.loading && this.email.length > 3 && this.email !== this.initialValue
```

Comment:
> this.email.length > 3

What happens when submit any string longer than 3 characters? What about making a email format check with a regex to be sure that it's a valid email?